src/runner.go

```diff
@@ -35,22 +35,17 @@ var verificationArgs = []string{
 
 // Verify that no one tampered with yaml file
 func verifyYamlFile(yamlData []byte) bool {
-
 	if !*config.VerifyYAML {
 		log.Warnln("WARNING: Playbook verification disabled.")
 		return true
 	}
 
-	// --payload here will be a no-op because no upload is performed when using the verifier
-	//   but, it will allow us to update the egg!
-
-	env := os.Environ()
-
 	if !*config.InsightsCoreGPGCheck {
-		verificationArgs = append(verificationArgs, "--no-gpg")
-		env = append(env, "BYPASS_GPG=True")
```

Comment:
but shouldn't these two lines stay here? If it is disabled then how does the insights-client knows it is?

I know I just did what they had in [worker-playbook](https://github.com/RedHatInsights/rhc-worker-playbook/blob/main/rhc_worker_playbook/server.py#L193 ), now it doesn't make sense to me, are those line unnecessary?

Reply:
Added both lines back.